addon/components/o-s-s/modal.stories.js

```diff
@@ -4,72 +4,108 @@ import { action } from '@storybook/addon-actions';
 export default {
   title: 'Components/OSS::Modal',
   component: 'modal',
-  args: {
-    title: 'Modal Title'
-  },
   argTypes: {
     title: {
-      control: 'text'
+      description: 'The title in the header of the modal',
+      table: {
+        type: {
+          summary: 'string'
+        },
+        defaultValue: { summary: 'undefined' }
+      },
+      control: { type: 'text' }
+    },
+    options: {
+      description:
+        'The options available:<br>' +
+        '- centered: boolean<br>' +
+        '- modalClass: string<br>' +
+        '- header: boolean<br>' +
+        '- borderlessHeader: boolean<br>' +
+        '- tabindex: number',
+      table: {
+        type: {
+          summary: 'object'
+        },
+        control: { type: 'object' }
+      }
     },
-    options: {}
+    onClose: {
+      description: 'A callback that sends when clicking on closing button',
```

Comment:
A callback triggered when the close button is clicked.

(I think this component is deprecated though)

Reply:
Yep, I will add a mention of that